docs/release-notes.rst

```diff
@@ -6,6 +6,7 @@ Release notes
 
 Release 0.10.1 (unreleased)
 ===========================
+- `PR 674 <https://github.com/uber/petastorm/pull/674>`_: tf_dataset: add unit test to verify repeat() works after cache().
```

Comment:
nit: usually I do not document unit-tests changes in release doc as this is not user facing (unless there is a bug fix).

Reply:
Will remove it.